docs/HikvisionNVRDoc.md

```diff
@@ -0,0 +1,377 @@
+## About the connector
+Hikvision's Network Video Recorders (NVRs) provide advanced artificial intelligence capabilities for any connected data stream, even those from conventional security cameras
+<p>This document provides information about the Hikvision NVR Connector, which facilitates automated interactions, with a Hikvision NVR server using FortiSOAR&trade; playbooks. Add the Hikvision NVR Connector as a step in FortiSOAR&trade; playbooks and perform automated operations with Hikvision NVR.</p>
+
+### Version information
+
+Connector Version: 1.0.0
+
+Contributors: Islam Baker
+
+Authored By: Fortinet CSE
```

Comment:
I think publisher should be Fortinet instead of Fortinet CSE

Reply:
This connector was submitted by the Fortinet CSE team, for this reason publisher is Fortinet CSE.